src/polar/PolarAngleAxis.tsx

```diff
@@ -22,20 +22,12 @@ import { selectAngleAxis, selectPolarViewBox } from '../state/selectors/polarAxi
 import { defaultPolarAngleAxisProps } from './defaultPolarAngleAxisProps';
 import { useIsPanorama } from '../context/PanoramaContext';
 import { svgPropertiesNoEvents, svgPropertiesNoEventsFromUnknown } from '../util/svgPropertiesNoEvents';
+import { RequiresDefaultProps, resolveDefaultProps } from '../util/resolveDefaultProps';
 
 const eps = 1e-5;
 const COS_45 = Math.cos(degreeToRadian(45));
 
-/**
- * These are injected from Redux, are required, but cannot be set by user.
- */
-interface PropsInjectedFromRedux {
-  cx?: number;
```

Comment:
These three are ignored, they were the "internal props not recommended to be used" in 2.x